edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -100,17 +100,31 @@ def __init__(self, ws: os.PathLike, package_path_list: Iterable[os.PathLike],
             package_path_packages[package_path] = \
                 [Path(p).parent for p in package_path.glob('**/*.dec')]
 
+        ignore_nested_packages = False
```

Comment:
should we put comments around this code block that reference this will be removed.

Reply:
Added in PR commit https://github.com/tianocore/edk2-pytool-library/pull/199/commits/faf600ea5c0691f6f30c5ffdc9650750c245a90c